README.md

```diff
@@ -40,7 +40,15 @@ Fabrix solves this problem by automating the rendering of React components from
 >
 > We are still actively developing the features. Tell us your idea or feedback on [discussions](https://github.com/fabrix-framework/fabrix/discussions).
 
-See the instruction at [the quick start](./packages/fabrix/README.md#quick-start) to use fabrix in your React app, or [online interactive editor](https://fabrix-framework.github.io/editor/) is available to see how fabrix works out without any setups.
```

Comment:
Update here to have two sections: editor and examples. 

Added a screenshot for fabrix editor in addition.

![image](https://github.com/user-attachments/assets/80ccb1ef-f1e7-4c45-b129-256b171fd73c)

Reply:
Not now, but I will add a link on this image to the YouTube video later.